macos/Runner/AppDelegate.swift

```diff
@@ -1,7 +1,7 @@
 import Cocoa
 import FlutterMacOS
 
-@NSApplicationMain
+@main
```

Comment:
Not sure why we have the native changes. Are they because of the latest version of flutter? If they were modified automatically then its fine

Reply:
yes it was modified automatically when i used start debuging on macos